hazelcast-sql/src/test/java/com/hazelcast/jet/sql/SqlClientCompactQueryTest.java

```diff
@@ -132,190 +120,6 @@ public void testQueryOnPrimitive() {
         assertThat(result).hasSize(5);
     }
 
-    @Test
-    @Ignore("see https://github.com/hazelcast/hazelcast/issues/19427")
-    public void testQueryOnObject() {
-        //To be able to run comparison methods on objects on the server we need the classes
-        assumeTrue(clusterHaveUserClasses);
-
-        HazelcastInstance client = factory.newHazelcastClient(clientConfig());
-        IMap<Integer, Object> map = client.getMap("test");
-        for (int i = 0; i < 10; i++) {
-            map.put(i, new NodeDTO(new NodeDTO(i), i));
-        }
-
-        client.getSql().execute("CREATE MAPPING " + "test" + '('
-                + "__key INTEGER"
-                + ", id INTEGER"
-                + ", child OBJECT"
-                + ") TYPE " + IMapSqlConnector.TYPE_NAME + ' '
-                + "OPTIONS ("
-                + '\'' + OPTION_KEY_FORMAT + "'='" + "int" + '\''
-                + ", '" + OPTION_VALUE_FORMAT + "'='" + COMPACT_FORMAT + '\''
-                + ", '" + OPTION_VALUE_COMPACT_TYPE_NAME + "'='" + "employee" + '\''
-                + ")");
-
-        SqlResult result = client.getSql().execute("SELECT id FROM test WHERE child = ?", new NodeDTO(new NodeDTO(5), 5));
-
-        SqlRow row = Iterators.getOnlyElement(result.iterator());
-        assertEquals(new Integer(5), row.getObject("id"));
-    }
-
-    @Test
-    public void testNestedCompactAsColumn() {
-        HazelcastInstance client = factory.newHazelcastClient(clientConfig());
-        IMap<Integer, NodeDTO> map = client.getMap("test");
-        for (int i = 0; i < 10; i++) {
-            map.put(i, new NodeDTO(new NodeDTO(i), i));
-        }
-
-        client.getSql().execute("CREATE MAPPING " + "test" + '('
-                + "__key INTEGER"
-                + ", id INTEGER"
-                + ", child OBJECT"
```

Comment:
can it be `child COMPOSED` and the test still work?

Reply:
It is not supported yet.Nested support will be added later.